client/me/concierge/main.js

```diff
@@ -119,4 +144,5 @@ export default connect( ( state, props ) => ( {
 	site: getSite( state, props.siteSlug ),
 	scheduleId: getConciergeScheduleId( state ),
 	userSettings: getUserSettings( state ),
+	localize,
```

Comment:
The correct way of using this `localize()` is to pass the component we want to inject the i18n functions as props as its argument, just like `connect()` from `react-redux`. In this case, it should be `localize( ConciergeMain)`. e.g. https://github.com/Automattic/wp-calypso/blob/master/client/components/domains/map-domain-step/index.jsx#L270

However, since there is nothing needed to be translated in this component, importing it or calling it here shouldn't be necessary.

I don't really know why `test-client` has never complained before this PR, but I think the right way to fix the error here is to remove the `i18n-calypso` mock code entirely. i.e. This part in the test:
```
 10 jest.mock( 'i18n-calypso', () => ( {
 11     localize: Comp => props => (
 12         <Comp
 13             { ...props }
 14             translate={ function( x ) {
 15                 return x;
 16             } }
 17         />
 18     ),
 19     numberFormat: x => x,
 20     translate: x => x,
 21 } ) );
 22
```

Reply:
> The correct way of using this localize() is to...

I agree. I'll remove it in the next pass.

> ...I think the right way to fix the error here is to remove the i18n-calypso mock code entirely...

I'll try that too. Thanks for the suggestions. :-)